aws-rds-dbcluster/src/main/java/software/amazon/rds/dbcluster/UpdateHandler.java

```diff
@@ -61,7 +64,7 @@ protected ProgressEvent<ResourceModel, CallbackContext> handleRequest(
                     "Resource is immutable"
             );
         }
-
+        callbackContext.timestampOnce(RESOURCE_UPDATED_AT, Instant.now());
```

Comment:
Please put it right before the modify call is being invoked. It seems to be taken out of the context here.

Reply:
Done. I put her before any possible modification that can happen before the actual modifyDBCluster call  like removeFromGlobalCluster that can generate failure event `could not be removed from global cluster`.